### ISSUE TITLE
Wii Classic Controller Pro Analog Stick Support

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dborth/snes9xgx#390]
```
What steps will reproduce the problem?
Using the analog stick on a Wii Classic Controller Pro.

What is the expected output? What do you see instead?
Input not detected by emulator.

Does this issue occur on SNES9x for Windows 1.51? If so, please report the
issue on the Snes9x forum.
N/A

What version are you using? On GameCube or Wii?
Wii (V. 4.1.5)

Are you using the official build (from the downloads page or auto-update)?
Yes

Please provide any additional information below.
Basically analog stick support for the Wii Classic Controller Pro needs to
be added.
```

Original issue reported on code.google.com by `Adria...@gmail.com` on 28 Dec 2009 at 4:19
